README.md

```diff
@@ -63,6 +63,8 @@ Documentation of the LSST Science Pipelines is at https://pipelines.lsst.io
         $ ingestImagesDecam.py /path/to/repo --filetype raw /path/to/raw/*.fits.fz
         $ ingestCalibs.py /path/to/repo/  --calib /path/to/calib/repo/ --calibType defect /path/to/calib/*fits --validity 999
```

Comment:
No, the intention is still to use ingestCalibs.py for ingesting biases, flats, fringes, and defects. The suggestion I was referring to above is the `--mode=link` suggestion explicitly; one should just leave `--mode` out of it altogether for fringes and defects. I think the latest commit below (which also includes an update to the README) clarifies this.